test/image/mocks/bar_attrs_group_norm.json

```diff
@@ -1,19 +1,24 @@
 {
     "data":[
         {
-            "base":4,
-            "x":[3,2,1,0],
+            "text":0.5,
+            "textposition":"outside",
+            "outsidetextfont": {"size":32},
```

Comment:
@etpinard This is on purpose to force a rotation in all the outside labels. If the font is small enough, the label can be drawn without rotation.